server.go

```diff
@@ -1573,9 +1568,7 @@ func (s *server) establishConn(sp *serverPeer) error {
 	if err != nil {
 		return err
 	}
-	if err := sp.Connect(conn); err != nil {
-		return err
-	}
+	sp.Connect(conn)
```

Comment:
If the connect fails, you will get both:
log.Errorf("Cannot start peer %v: %v", p, err)
srvrLog.Debugf("Connected to %s", sp.Addr())